browser/ui/webui/brave_wallet/wallet_page_ui.cc

```diff
@@ -32,7 +32,9 @@
 #include "brave/components/ipfs/buildflags/buildflags.h"
 #include "brave/components/l10n/common/localization_util.h"
 #include "chrome/browser/profiles/profile.h"
+#if !BUILDFLAG(IS_ANDROID)
 #include "chrome/browser/ui/webui/sanitized_image_source.h"
```

Comment:
I see that file is excluded for Android, most likely we can get rid of it on a swap page as well and remove a non Android include in gn for it.

Reply:
fixed (removed changes in the wallet_page_ui.cc because it is not involved in the swap page loading)